csharp/src/SeedLang.Common/Severity.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace SeedLang.Common {
+  // Common severity levels for diagnostics, logging info, etc.
+  public enum Severity {
```

Comment:
How about reversing the order, so that the big number indicates more serious error after serialization?

Reply:
Done. Also assigned explicit integer values to them.